includes/CMB2_Options_Hookup.php

```diff
@@ -50,6 +50,17 @@ public function hooks() {
 			add_filter( "cmb2_should_autoload_{$this->option_key}", '__return_false' );
 		}
 
+		/**
+		 * for WP < 4.7
+		 *
+		 * @see https://core.trac.wordpress.org/ticket/37885
+		 */
+		global $wp_version;
+		if ( version_compare( $wp_version, '4.7', '<' ) ) {
```

Comment:
Let's use `if ( ! CMB2_Utils::wp_at_least( '4.7' ) && ! function_exists( 'register_setting' ) ) {`, but this looks good.

Reply:
yes, that's off course better :)

Reply:
Yes, I tested it for all: WP < 4.7, = 4.7 > 4.7.